lock-impl/src/main/java/com/palantir/lock/logger/SimpleLockRequest.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.lock.logger;
+
+import javax.annotation.Nullable;
+
+import org.immutables.value.Value;
+
+import com.palantir.lock.LockRequest;
+import com.palantir.lock.TimeDuration;
+
+@Value.Immutable
+public abstract class SimpleLockRequest {
+
+    public static SimpleLockRequest of(LockRequest request, String lockDescriptor, String clientId) {
```

Comment:
Turns out this is part of hiding the descriptor that would otherwise be exposed when serializing the LockRequest.